### PR TITLE
Increase timeouts for debugger-related selenium tests to make them stable

### PR DESCRIPTION
**Pull request by dmytro-ndp**
### What does this PR do?
It increases timeouts in the **CodenyEditor** and **DebugPanel** page objects in methods related to debugger (from 10/20 seconds to 60) to avoid failures like [the follow](https://ci.codenvycorp.com/view/qa/job/che-integration-tests-master-docker/195/testReport/org.eclipse.che.selenium.debugger/StepIntoStepOverStepReturnWithChangeVariableTest/changeVariableTest/):
```
java.lang.AssertionError: Known issue: https://github.com/eclipse/che/issues/8105
	at org.testng.Assert.fail(Assert.java:83)
	at org.eclipse.che.selenium.debugger.StepIntoStepOverStepReturnWithChangeVariableTest.changeVariableTest(StepIntoStepOverStepReturnWithChangeVariableTest.java:149)
...
Caused by: org.openqa.selenium.TimeoutException: Expected condition failed: waiting for presence of any elements located by By.xpath: //div[@id='gwt-debug-editorPartStack-contentPanel']//div[@active]//div[@class='textviewContent' and @contenteditable='true']//span[@debugid='debug-line'] (tried for 20 second(s) with 500 MILLISECONDS interval)
Build info: version: '3.0.1', revision: '1969d75', time: '2016-10-18 09:49:13 -0700'
System info: host: 'slave2.codenvycorp.com', ip: '127.0.0.1', os.name: 'Linux', os.arch: 'amd64', os.version: '3.10.0-514.16.1.el7.x86_64', java.version: '1.8.0_102'
Driver info: org.eclipse.che.selenium.core.SeleniumWebDriver
	at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:80)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:265)
	at org.eclipse.che.selenium.pageobject.debug.DebugPanel.waitDebugHighlightedText(DebugPanel.java:359)
```

It also fixes **MachineTerminal.launchScriptAndGetInfo()** method.

### What issues does this PR fix or reference?
#8105

### CODE REVIEW

**Comment by dmytro-ndp** [comment]
ci-test

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/machineperspective/MachineTerminal.java [hunk 1]
```diff
@@ -322,42 +321,21 @@ private String getTerminalTabXPath(int terminalNumber) {
...
-      String webAppPIDs = consoles.getVisibleTextFromCommandConsole();
-      LOG.warn("@@@ The PID list from run Web application is: " + webAppPIDs);
+      commandsPalette.startCommandByDoubleClick(getApplicationInfoCommandName);
+
+      String applicationInfo = consoles.getVisibleTextFromCommandConsole();
+      LOG.warn("@@@ Web application info: {}", applicationInfo);
     } catch (Exception ex) {
-      LOG.error("@@@ Cannot catch the info about PID", ex);
+      LOG.error("@@@ Cannot catch the info about application.", ex);
     }
```
Comment (musienko-maxim) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/machineperspective/MachineTerminal.java:339]: Do we still need get info. about PIDs? I think we can remove this check

Comment (dmytro-ndp) [reply selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/machineperspective/MachineTerminal.java:339]: I would hold information about the web application in logs until debugger tests are stabilized.

**Review by musienko-maxim — APPROVED** [review]

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/machineperspective/MachineTerminal.java (hunk 1 above)
Comment (musienko-maxim) [reply selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/machineperspective/MachineTerminal.java:339]: ok

**Pull request merged**